### PR TITLE
[infra] Support extra arguments for llvm-cov + update the binaries.

### PR DESCRIPTION
**Pull request by Dor1s**
Requested by folks working on libaom.

### CODE REVIEW

File: infra/base-images/base-runner/coverage [hunk 1]
```diff
@@ -23,7 +23,10 @@ else
 fi
 
 LOGS_DIR="$OUT/logs"
-mkdir -p $LOGS_DIR
```
Comment (johannkoenig) [inline infra/base-images/base-runner/coverage]: Would it be worth deleting the entire $OUT directory?

File: docs/code_coverage.md [hunk 2]
```diff
@@ -72,6 +72,17 @@ via `--corpus-dir` to be used instead of the corpus downloaded from OSS-Fuzz:
...
 ```
 
+### Additional arguments for `llvm-cov`
+
+You might want to use some of the options of [llvm-cov tool] such as
+`-ignore-filename-regex`. You can pass those to the helper script via
+`--extra-args` flag:
+
+```bash
+python infra/helper.py profile --extra-args='-ignore-filename-regex=.*code/to/be/ignored/.*' $project_name
```
Comment (oliverchang) [inline docs/code_coverage.md]: I think a common convention with command line flags is to use "--" to denote that everything afterwards should be forwarded to a target program. 

I think argparse should support this also: https://stackoverflow.com/questions/14693718/how-to-parse-positional-arguments-with-leading-minus-sign-negative-numbers-usi

File: infra/base-images/base-runner/coverage (hunk 1 above)
Comment (Dor1s) [reply infra/base-images/base-runner/coverage]: No, there must be fuzzer binaries which are needed for generating code coverage report.

File: docs/code_coverage.md (hunk 2 above)
Comment (Dor1s) [reply docs/code_coverage.md]: That's an excellent suggestion, thanks Oliver! Done.

**Comment by Dor1s** [comment]
Please take another look :)

**Comment by Dor1s** [comment]
Landing this for now just to give something usable to libaom folks. If there are any other comments, we'll be happy to address in a follow-up.

**Pull request merged**